### DISCUSSION TITLE
I have a question about turborepo caching.

### DISCUSSION THREAD

**Discussion by rlaqudwls25** [vercel/turborepo#7100] [Help] [answered]
I understand that the build cache part is based on workspaces.

As shown in the screenshot, there are pages a and b, but if only a is modified and built using ssg, b has not been modified, so I am trying to use the part that is already cached.
Is build caching not possible on a per-page basis?

<img width="419" alt="스크린샷 2024-01-25 오후 6 45 28" src="https://github.com/vercel/turbo/assets/65752350/f46bc109-b1c9-4392-bd77-e5e224b851e0">

**Comment by anthonyshew** [comment, marked as answer]
Turborepo caches at the package-level so this sort of behavior isn't planned for Turborepo. There is a long-term vision for Turbopack to have the level granularity you're describing!
